JSDemos/Demos/List/ListEditingAndAPI/description.md

```diff
@@ -1 +1,26 @@
-The **List** is&nbsp;a&nbsp;UI component that represents a&nbsp;collection of&nbsp;items in&nbsp;a&nbsp;scrollable list. This demo displays different delete modes and shows how you can handle the **selectionChanged** event.
\ No newline at end of file
+The [List](/Documentation/ApiReference/UI_Widgets/dxList/) displays a collection of items as a scrollable list. To create a list, pass an array of items to **dataSource**. 
+
+This demo illustrates deletion. The UI consists of the **List** and the "Options" section. 
+
+To allow users delete items, set the [allowItemDeleting](/Documentation/ApiReference/UI_Widgets/dxList/Configuration/#allowItemDeleting) property to *true*. Use the checkbox in the "Options" section to see how it works.
+
+The **List** supports three default and three additional deletion modes. You can see them in action in the "itemDeleteMode" dropdown menu of the "Options" section.
+
+**Default modes**
```

Comment:
The grouping is unnecessary. We can indicate that one or another mode is default for a platform like this:

`- *"slideItem"* (default for iOS)`

Let's get back to a plain list.